src/lang.js

```diff
@@ -27,7 +27,7 @@
     // Default options //
 
     var defaults = {
-        defaultLocale: 'en' /** The default locale if not set. */
+        defaultLocale: 'en', /** The default locale if not set. */
```

Comment:
Sorry @rmariuzzo i didn't notice that :(. I will be using jshint for future commits.